JSONFormat.go

```diff
@@ -39,7 +45,7 @@ func (j jsonEntry) MarshalJSON() ([]byte, error) {
 		return nil, ErrNoMessage
 	}
 
-	_, err := buf.WriteString(fmt.Sprintf("{\"%s\":\"%s\", \"%s\":\"%s\", \"%s\":\"%s\"", j.levelKey, j.Level, j.eventKey, j.Event, j.messageKey, j.Message))
+	_, err := buf.WriteString(fmt.Sprintf("{\"%s\":\"%s\", \"%s\":\"%s\", \"%s\":\"%s\"", j.levelKey, j.Level, j.eventKey, j.Event, j.messageKey, escapeSpecialChars(j.Message)))
```

Comment:
Go is low level, don't need anything but the standard library :P
Personally, too high level and you go into "magic" behavior and I hate magic.

Pragmatically speaking, this code works and you would create new data structures similar to that, and then rely in the default MarshallJson interface, replacing some rather simple code with more data and more complex code, albeit shorter, just so that you would avoid string formatting?

It's up to you guys, after all I'm not gonna maintain this. For me, the less magic, the dumber the code, the better.